app/views/application/_footer.html.erb

```diff
@@ -9,8 +9,8 @@
         <li class="mb-2"><%= t ".by_html" %></li>
         <li class="mb-2"><%= t ".open_source_html" %></li>
         <li class="mb-2"><%= t ".version" %>:
-          <%= link_to "#{ENV.fetch("APP_VERSION", "unknown").split(":")[-1]} (#{ENV.fetch("GIT_SHA", "main")[0, 7]})",
-                "https://github.com/manyfold3d/manyfold/tree/#{ENV.fetch("GIT_SHA", "main")}" %>
+          <%= link_to "#{Rails.application.config.app_version} (#{Rails.application.config.git_sha})",
```

Comment:
```suggestion
          <%= link_to "#{Rails.application.config.app_version} (#{Rails.application.config.git_sha}[0, 7])",
```

Cut down the git sha when displayed to the first 8 chars, otherwise it'll be huge